lib/shared_print/groups.rb

```diff
@@ -0,0 +1,88 @@
+module SharedPrint
+  # A lookup object for which orgs are members of which shared print program,
+  # and vice versa.
+  class Groups
+    attr_reader :groups
+
+    def initialize
+      @groups = {
```

Comment:
If it all possible I would extract this `@groups` data into a JSON or YAML config-ish file and read it in.

Reply:
In general I'd agree with this, but the SharedPrint::Groups is such a specific bit of data that I would like to have committed anyways, and I don't foresee a lot of changes to it. I'm perfectly happy hardcoding this (famous last words).